Source/Core/BoogiePL.atg

```diff
@@ -1110,8 +1115,24 @@ LabelOrCmd<out Cmd c, out IToken label>
      c = dummyCmd;  label = null;
      Cmd/*!*/ cn;
      QKeyValue kv = null;
+     bool isHide;
+     IdentifierExpr hideRevealId = null;
   .)
-  ( LabelOrAssign<out c, out label>
+  ( (
+      ( "reveal" (. isHide = false; .) 
+      | "hide"   (. isHide = true;  .)
+      )
+      ( ident    (. hideRevealId = new IdentifierExpr(t, t.val); .)
+      | "*"
+      )
+      (. c = hideRevealId == null ? new HideRevealCmd(t, isHide) : new HideRevealCmd(hideRevealId, isHide); .)
+      ";"
+    )
+  | LabelOrAssign<out c, out label>
+  | "popScope"            (. c = new ChangeScope(t, false); .)
```

Comment:
I don't think anything else in Boogie, the language, uses camel case like this. I'd maybe prefer just `push` and `pop`. The word `Scope` doesn't really tell you what kind of scope it is, so I'm not sure it adds much.